src/main/java/uk/gov/beis/els/categories/spaceheaters/model/BoilerCombinationCalculatorForm.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.beis.els.categories.spaceheaters.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import uk.gov.beis.els.api.common.ApiValuesFromLoadProfileList;
+import uk.gov.beis.els.categories.waterheaters.service.WaterHeatersService;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+
+public class BoilerCombinationCalculatorForm extends BoilerPackagesCalculatorForm {
+
+  @FieldPrompt("Water heating energy efficiency of water heater (%)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the water heating efficiency (%), up to 3 digits long")
+  @Schema(type = "integer")
```

Comment:
I left the schema values from the water heaters form just to make it quicker to make the new API endpoints